src/components/testCaseLanding/common/TestCaseTable/TestCaseTable.tsx

```diff
@@ -91,6 +91,32 @@ const TestCaseTable = (props: TestCaseTableProps) => {
     action: any;
     id: string;
   };
+
+  function customSortGroup(rowA, rowB) {
+    const a = rowA.original.group;
+    const b = rowB.original.group;
+    if (a === undefined || a === "") {
+      return 1;
+    } else if (b === undefined || b === "") {
+      return -1;
+    }
+    if (a < b) return -1;
+    if (a > b) return 1;
+    return 0;
+  }
+  function customSortDesc(rowA, rowB) {
+    const a = rowA.original.description;
+    const b = rowB.original.description;
+    if (a === undefined || a === "") {
+      return 1;
+    } else if (b === undefined || b === "") {
+      return -1;
+    }
+    if (a < b) return -1;
+    if (a > b) return 1;
+    return 0;
+  }
```

Comment:
I don't think there is any difference between these two functions. Can we not reuse one?